plugins/include/sqlx.inc

```diff
@@ -105,171 +129,248 @@ native SQL_QuoteString(Handle:db, buffer[], buflen, const string[]);
- * A successful query starts at the first result,
- *  so you should not call SQL_NextRow() first.
- * Passing no extra params - return int
- * Passing one extra param - return float in 1st extra arg
- * Passing two extra params - return string in 1st arg, max length in 2nd
- * Example:
- *  new num = SQL_ReadResult(query, 0)
- *  new Float:num2
- *  new str[32]
- *  SQL_ReadResult(query, 1, num2)
- *  SQL_ReadResult(query, 2, str, 31)
+ * A successful query starts at the first result, so you should not call
+ * SQL_NextRow() first.
+ * 
+ * @note Example how to get different types of values:
+ *       new num = SQL_ReadResult(query, 0)
+ *       new Float:num2
+ *       new string[32]
+ *       SQL_ReadResult(query, 1, num2)
+ *       SQL_ReadResult(query, 2, str, 31)
+ *
+ * @param query         Handle of a query to read results from.
+ * @param column        Which column to get the value from.
+ * @param ...           Passing no extra arguments - returns an integer.
+ *                      Passing one extra argument - returns a float in the first extra argument
+ *                      Passing two extra params - returns a string in the first argument
+ *                      with a maximum string length in the second argument.
+ * 
+ * @return              If no extra arguments are passed, returns an integer value.
  */
 native SQL_ReadResult(Handle:query, column, any:...);
 
 
 /**
- * Advances to the next result (return value should be ignored).
+ * Advances to the next result (row).
+ *
+ * @param query         Handle of a query.
+ * 
+ * @noreturn
  */
 native SQL_NextRow(Handle:query);
 
 
-/** 
- * Returns the number of affected rows.
+/**
+ * Returns the number of affected rows by a query.
+ *
+ * @param query         Handle of a query to check.
+ * 
+ * @return              The number of affected rows.
  */
 native SQL_AffectedRows(Handle:query);
 
 
 /**
- * Returns the number of rows total.
+ * The number of retrieved rows (results) after a query.
+ *
+ * @param query         Handle of a query to check.
+ * 
+ * @return              The number of retrieved rows by the query.
  */
 native SQL_NumResults(Handle:query);
 
 
 /**
- * Returns the number of columns total.
+ * Returns the total number of columns.
+ *
+ * @param query         Handle of a query to check.
+ * 
+ * @return              The number of retrieved columns by the query.
  */
 native SQL_NumColumns(Handle:query);
 
 
-/** 
- * Returns the name of a column.
- * Errors on a bad field number.
+/**
+ * Retrieves the name of a column by its index.
+ *
+ * @param query         Handle of a query.
+ * @param num           The number (index) of a column to retrieve the name from.
+ * @param name          Buffer where to store the column's name.
+ * @param maxlength     Maximum length of the output buffer.
+ * 
+ * @noreturn
+ * @error               If a bad column index is supplied.
  */
 native SQL_FieldNumToName(Handle:query, num, name[], maxlength);
```

Comment:
Misses `@error` with those possibles errors `Invalid query handle`, `No result set in this query`